package.json

```diff
@@ -23,37 +23,37 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
-    "URIjs": "1.14.1",
-    "bluebird": "2.3.10",
+    "URIjs": "^1.14.2",
```

Comment:
My understanding of shrinkwrap is that it wraps the versions that are
**currently** installed in the node_modules directory. So running it again
at a later time won't adjust the shrinkwrap, other then if you've
intentionally `npm install foo@newer`.

On Fri, Mar 27, 2015 at 4:45 PM, Vlad Filippov <notifications@github.com>
wrote:

> In package.json
> <https://github.com/mozilla/fxa-oauth-server/pull/237#discussion_r27338856>
> :
>
> > @@ -23,37 +23,37 @@
> >      "node": ">=0.10.0"
> >    },
> >    "dependencies": {
> > -    "URIjs": "1.14.1",
> > -    "bluebird": "2.3.10",
> > +    "URIjs": "^1.14.2",
>
> I think it won't take care of it if someone adds a new (dev)dependency and
> rewraps it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/fxa-oauth-server/pull/237/files#r27338856>.
>